### ISSUE TITLE
ManagedCertification is not created

### ISSUE THREAD

**Issue by TrsNium** [GoogleCloudPlatform/gke-managed-certs#22]
Hello.

I deploy Managed Certificate custom object. 
But Managed Certificate is not created.

My kubernetes version is following.
`v1.12.7-gke.10`

Deployed Managed Certificate and ingress is following.
```
apiVersion: networking.gke.io/v1beta1
kind: ManagedCertificate
metadata:
  name: cert
spec:
  domains:
    - test.hoge.com
---
apiVersion: extensions/v1beta1
kind: Ingress
metadata:
  name: ingress
  annotations:
    kubernetes.io/ingress.global-static-ip-name: "ingress-ip"
    networking.gke.io/managed-certificates: "cert"
spec:
  backend:
    serviceName: test
    servicePort: 80
```

if you have already solution, please teach me.
Thank you.

**Comment by krzykwas** [comment]
If your cluster is regional, this is not working because of a bug in 1.12.7-gke.10, but the bug is fixed in 1.12.7-gke.17.

**Comment by TrsNium** [comment]
I see! Thank you for the useful information.
